client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -39,44 +39,41 @@ const Unlocked = React.createClass( {
 
 				switch ( error.error ) {
 					case 'enable_private_reg_failed':
-						errorMessage = this.translate( 'We were unable to enable Privacy Protection for your domain. ' +
+						errorMessage = translate( 'We were unable to enable Privacy Protection for your domain. ' +
 							'Please try again or {{contactLink}}Contact Support{{/contactLink}} if you continue to have trouble.',
 							{ components: { contactLink } } );
 						break;
 					case 'decline_transfer_failed':
-						errorMessage = this.translate( 'We were unable to stop the transfer for your domain. ' +
+						errorMessage = translate( 'We were unable to stop the transfer for your domain. ' +
 							'Please try again or {{contactLink}}Contact Support{{/contactLink}} if you continue to have trouble.',
 							{ components: { contactLink } } );
 						break;
 					case 'lock_domain_failed':
-						errorMessage = this.translate( 'We were unable to lock your domain. ' +
+						errorMessage = translate( 'We were unable to lock your domain. ' +
 							'Please try again or {{contactLink}}Contact Support{{/contactLink}} if you continue to have trouble.',
 							{ components: { contactLink } } );
 						break;
 					default:
-						errorMessage = this.translate(
+						errorMessage = translate(
 							'Oops! Something went wrong and your request could not be ' +
 							'processed. Please try again or {{contactLink}}Contact Support{{/contactLink}} if ' +
 							'you continue to have trouble.', { components: { contactLink } }
 						);
 						break;
 				}
 				notices.error( errorMessage );
+
+				this.setState( { submitting: false } );
 			} else if ( hasPrivacyProtection ) {
-				notices.success( this.translate( 'We\'ve canceled your domain transfer. Your domain is now locked and ' +
+				notices.success( translate( 'We\'ve canceled your domain transfer. Your domain is now locked and ' +
```

Comment:
Same comment as above about global notices. If you prefer not to do it in this PR, that's cool, but it's not much more work - just pass the actions to `connect` and replace the call here.